reset-pass/account-validator-tool/src/webapp/css/passwordReset.css

```diff
@@ -169,27 +171,28 @@ table.myform {
 
 #strengthInfo
 {
-	display: block;
-	color: #000;
-	background-color: #fff;
-	border: 1px solid #ccc;
-	border-radius: 5px;
-	box-shadow: 1px 2px 3px rgba(0, 0, 0, 0.2);
-	left: 34em;
-	padding: 1em;
-	position: absolute;
-	margin-top: -5em;
-	width: 50%;
+    display: block;
```

Comment:
Some lines in this section are unchanged except for whitespace. While we're not overly concerned with that around here, the Sakai community frowns on it. Just be aware of this if you are submitting PRs that are intended to also be contributed back to Sakai rather than just for OWL.